payment/key.go

```diff
@@ -1,18 +1,16 @@
 package payment
 
 import (
-	"crypto/aes"
-	"crypto/cipher"
 	"crypto/rand"
 	"encoding/base64"
 	"encoding/hex"
 	"fmt"
-	"os"
 	"time"
+
+	cryptography "github.com/brave-intl/bat-go/utils/cryptography"
 )
 
-// AESKey for encrypting secrets
-var AESKey = os.Getenv("ENCRYPTION_KEY")
+// What the merchant key length should be
 var keyLength = 24
```

Comment:
is this the secretbox key? it should be 32 like in https://github.com/brave-intl/bat-go/pull/347/files#diff-abc59134b74da5d26a12093a0ab876a5R17 if so

Reply:
nvm, i guess the merchant keys aren't secretbox? (what are they used for?)

Reply:
Yep, this is a little bit confusing 😅  The API Key length for the merchant is 24 characters long. We can change this to be 32 characters if it would make it more secure / easier to understand.

Reply:
ah got it. it depends a bit on how the API key is used. is it essentially just an auth token, or does it get used for encryption/signing somewhere? is there some rate limit on how often an attacker can guess and check tokens?

Reply:
Right now the key is used to sign a MAC for the merchant. For verifying tokens there is a rate limit on the request, and if there is multiple tokens sent in one request we verify and return a `400` if any token is invalid.

Reply:
presumably it gets converted to 32bytes in https://github.com/go-macaroon/macaroon/blob/v2.1.0/crypto.go#L30 so i think 24 is fine here